### ISSUE TITLE
Change vibrancy from light to appearance-based

### ISSUE THREAD

**Issue by jonjesbuzz** [sindresorhus/caprine#648]
The vibrancy for "light" makes text unreadable on certain backgrounds in macOS Mojave. Consider using appearance-based instead of light, which makes it easier to read.

**Comment by sindresorhus** [comment]
Yes, that's the plan, but right now we're limited by what Electron provides. 

This is blocked by https://github.com/electron/electron/issues/13175.
